### DISCUSSION TITLE
Can the federation still be used now?

### DISCUSSION THREAD

**Discussion by lty628** [minio/minio#19355] [Q&A] [answered]
I have not been successful in configuring based on this document. Is there an issue with my configuration, or is federated mode no longer supported at all? 
Document address: https://github.com/minio/minio/blob/master/docs/federation/lookup/README.md

**Comment by harshavardhana** [comment, marked as answer]
It can be but not recommended. We haven't found any user who has the scale to use it appropriately.

What kind of scale are you looking for?

**Reply by Clara12062** [reply]
Hi

I also have the same confusion.

Can federated mode handle scaling of minio at the replica level? 

Now I could only expand the serverpool first, then use `decommission`, and then delete the original serverpool in cluster. This series of operations is not easy to automate, because decommission is not a simple operation.

**Reply by klauspost** [reply]
@Amber-976 Reach out on `sales@min.io` for a more in-depth discussion, describing your case.
